opensearch_py_ml/ml_commons/validators/stats.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+"""Module for validating Stats API parameters """
+
+
+def validate_stats_input(node_id, stat_id, payload):
```

Comment:
I feel like creating separate file for every methods is an overkill?

Reply:
may be we can merge all these validator methods into a common file?

Reply:
yes, I did think of it. I already am doing that in a separate branch. Will raise that separately in sometime.